tests/test_models.py

```diff
@@ -1,101 +1,37 @@
-    from inflammation.models import daily_max
-    npt.assert_array_equal(daily_max(np.array(test)), np.array(expected))
-
-@pytest.mark.parametrize(
-    "test, expected",
-    [
-        ([ [0, 0, 0], [0, 0, 0], [0, 0, 0] ], [0, 0, 0]),
-        ([ [4, 2, 5], [1, 6, 2], [4, 1, 9] ], [1, 1, 2]),
-        ([ [4, -2, 5], [1, -6, 2], [-4, -1, 9] ], [-4, -6, 2]),
-    ])
-def test_daily_min(test, expected):
-    """Test min function works for zeroes, positive integers, mix of positive/negative integers."""
-    from inflammation.models import daily_min
-    npt.assert_array_equal(daily_min(np.array(test)), np.array(expected))
-
-@pytest.mark.parametrize(
-    "test, expected, expect_raises",
-    [
-        (
-            [[0, 0, 0], [0, 0, 0], [0, 0, 0]],
-            [[0, 0, 0], [0, 0, 0], [0, 0, 0]],
-            None
-        ),
-        (
-            [[1, 1, 1], [1, 1, 1], [1, 1, 1]],
-            [[1, 1, 1], [1, 1, 1], [1, 1, 1]],
-            None
-        ),
-        (
-            [[float('nan'), 1, 1], [1, 1, 1], [1, 1, 1]],
-            [[0, 1, 1], [1, 1, 1], [1, 1, 1]],
-            None
-        ),
-        (
-            [[1, 2, 3], [4, 5, float('nan')], [7, 8, 9]],
-            [[0.33, 0.67, 1], [0.8, 1, 0], [0.78, 0.89, 1]],
-            None
-        ),
-        (
-            [[-1, 2, 3], [4, 5, 6], [7, 8, 9]],
-            [[0, 0.67, 1], [0.67, 0.83, 1], [0.78, 0.89, 1]],
-            ValueError,
-        ),
-        (
-                'hello',
-                None,
-                TypeError,
-        ),
-        (
-                3,
-                None,
-                TypeError,
-        ),
-        (
-            [[1, 2, 3], [4, 5, 6], [7, 8, 9]],
-            [[0.33, 0.67, 1], [0.67, 0.83, 1], [0.78, 0.89, 1]],
-            None,
-        )
-    ])
-def test_patient_normalise(test, expected, expect_raises):
-    """Test normalisation works for arrays of one and positive integers."""
-    from inflammation.models import patient_normalise
-
-    if isinstance(test, list):
-        test = np.array(test)
-
-    if expect_raises is not None:
-        with pytest.raises(expect_raises):
-          result = patient_normalise(test)
-          npt.assert_allclose(result, np.array(expected), rtol=1e-2, atol=1e-2)
-
-    else:
-        result = patient_normalise(test)
-        npt.assert_allclose(result, np.array(expected), rtol=1e-2, atol=1e-2)
\ No newline at end of file
+    npt.assert_array_equal(daily_mean(test_input), test_result)
+def test_daily_mean_integers():
+    """Test that mean function works for an array of positive integers."""
+    from inflammation.models import daily_mean
+    test_input = np.array([[1, 2],
+                           [3, 4],
+                           [5, 6]])
+    test_result = np.array([3, 4])
+    # Need to use Numpy testing functions to compare arrays
+    npt.assert_array_equal(daily_mean(test_input), test_result)
+
+
+@pytest.mark.parametrize('data, expected_standard_deviation', [
```

Comment:
Consider adding some tests for:
- Multiple patients 
- Validity of the function in case of unexpected values (negative, strings, etc.)
- Test to see if std is actually plotted.